docs/source/index.rst

```diff
@@ -66,6 +66,14 @@ flow of data.  Typically people rely on tools like ``itertools.tee``, and
 However this quickly become cumbersome, especially when building complex
 pipelines.
 
+Installation
+------------
+
+To install either use:
+
+- conda-forge: ``conda install streamz -c conda-forge``
+- pip: ``python -m pip install streazm``
+- local pip: ``git clone git@github.com:python-streamz/streamz.git; python -m pip install -e streamz``
```

Comment:
`pip install git+https://github.com/python-streamz/streamz` ?

Reply:
Does that install a local editable copy? (I'm mostly thinking of future streamz developers here)

Reply:
No, it uses a temporary copy.
Instead, then, I would say
```
- dev: git clone https://github.com/python-streamz/streamz
       pip install -e streamz/
```